components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -799,9 +799,25 @@ function ImageBuildView(props: ImageBuildViewProps) {
                 <WorkspaceLogs logsEmitter={logsEmitter} errorMessage={props.error?.message} />
             </Suspense>
             {!!props.onStartWithDefaultImage && (
-                <button className="mt-6 secondary" onClick={props.onStartWithDefaultImage}>
-                    Continue with Default Image
-                </button>
+                <>
+                    <div className="mt-6 w-11/12 lg:w-3/5">
+                        <p className="text-center text-gray-400 dark:text-gray-500">
+                            <span className="font-bold">Did you know?</span> You can use <code>gp rebuild</code> to
+                            validate and debug your custom workspace image before pushing &nbsp;&middot;&nbsp;
+                            <a
+                                href="https://www.gitpod.io/docs/configure/workspaces/workspace-image#trying-out-changes-to-your-dockerfile"
```

Comment:
💭 **_Unrelated to this PR_**: Can we mention about `gp rebuild` on [this page](https://www.gitpod.io/docs/references/gitpod-cli) also? _(if yes, I can take this up 🙋‍♂️)_

Reply:
@Siddhant-K-code `gp rebuild` is already part of that page 😄

Reply:
Oh Ok. I just saw the https://github.com/gitpod-io/website/pull/3226 🤦‍♂️